### ISSUE TITLE
Automatic release of phar file from Travis

### ISSUE THREAD

**Issue by wysow** [CouscousPHP/Couscous#81]
I have started something on my own fork about Travis releases here: https://github.com/wysow/Couscous

Look at the `master` branch for the last commits not included in original Couscous repo to understand what I've made so far...

We can discuss about it here if needed.

**Comment by mnapoli** [comment]
Awesome! I see the Phar is uploaded in the Github release, that's really good stuff.

However http://couscous.io/couscous.phar will not exist anymore, which is too bad because people could have script to download it (though Couscous is still very new, so maybe not). Or people could hope to write scripts to download it, and having a dynamic URL will prevent that.

Here is an idea: we could write a `before` script (set in `couscous.yml`) that would download the highest stable version of couscous and put it in the `website/` directory? That way anytime the website is generated, couscous.phar will be refresh to be the latest stable.

**Comment by wysow** [comment]
Yes about the link to the phar file in documentation I've made a little javascript that keep it's url up to date, but you're right your solution sounds good and I going to work in this way.

**Comment by wysow** [comment]
@mnapoli I just discovered this blog post that can help us maybe... http://moquet.net/blog/distributing-php-cli/

What do you think?

**Comment by mnapoli** [comment]
There's some pretty interesting stuff in there. I'm +1 with trying anything suggested in there. Self-update would be really cool, and keeping the version number up to date is also really nice. But those should probably be worked in separate issues. I didn't see anything that could help with our deployment.

**Comment by wysow** [comment]
@mnapoli maybe we should "merge" this with #149 in some way no?

**Comment by mnapoli** [comment]
@wysow can we first merge #149? We can go step by step if #149 works today that's already a great improvement, and then we can think about this issue without blocking #149? Let me know.

**Comment by wysow** [comment]
Yes sure we can go this way and see how we can improve the #149 mechanism...
